README.md

```diff
@@ -7,10 +7,23 @@ Creates the storage infrastructure required to use an instance of the HybridStor
 
 ## Usage
 
+You can either use a `hash-store` or `hash-range-store` implementation, which must
+match the type of store you're using in your application.
```

Comment:
Maybe worth explaining a bit more about what the difference is? e.g. “you can store multiple versions in one, only the latest version in the other”

Reply:
Would this explanation best live in the library and we link through to there?